src/js/index.js

```diff
@@ -104,3 +105,25 @@ window.addEventListener('load', () => {
 
 	cascade.call(window, true);
 });
+
+function setTheme(preference) {
+	if (preference === 'dark') {
+		document.body.classList.add('dark');
+		colorSwitch.innerHTML = 'Light mode';
```

Comment:
I think it would be better to set `colorSwitch.textContent` here since the value used is plaintext and doesn't contain any HTML.